contexts/TokenContext.tsx

```diff
@@ -27,6 +30,12 @@ export const TokenListProvider = ({ children }: { children: ReactNode }) => {
     error: balanceError,
     data,
   } = useTokenBalances({ tokenList: tokenList || [] });
+  const {
+    isLoading: allowanceIsLoading,
+    error: allowanceError,
```

Comment:
`allowanceIsLoading` and `allowanceError` are never used, but seems useful to log if they return an error below like we do for token balances

Reply:
These should be used now